tests/src/FunctionalJavascript/Form/AuthenticationFormJsTest.php

```diff
@@ -141,6 +141,36 @@ public function testKeyAddForm() {
     $this->validateForm([$this, 'visitKeyAddForm']);
   }
 
+  /**
+   * Tests the Authentication form using Hybrid auth.
+   *
+   * @group hybrid
+   */
+  public function testUsingHybridForm() {
+    $organization = getenv('APIGEE_EDGE_HYBRID_ORGANIZATION');
+    $account_key = getenv('APIGEE_EDGE_ACCOUNT_JSON_KEY');
+
+    if (!$organization || !$account_key) {
+      $this->markTestSkipped('Skipping "testUsingHybridForm": missing test environment variables APIGEE_EDGE_HYBRID_ORGANIZATION and/or APIGEE_EDGE_ACCOUNT_JSON_KEY.');
+    }
+
+    $web_assert = $this->assertSession();
+
+    // Test the authentication form.
+    $this->drupalLogin($this->rootUser);
+    $this->drupalGet(Url::fromRoute('entity.key.add_form'));
+    $this->visitKeyAddForm();
+
+    $page = $this->getSession()->getPage();
+
+    $page->selectFieldOption('key_input_settings[instance_type]', EdgeKeyTypeInterface::INSTANCE_TYPE_HYBRID);
+    $page->fillField('Organization', $organization);
+    $page->fillField('Account key', $account_key);
```

Comment:
Make sure to update this line if you take my field title change above

Reply:
Updated field name :+1: